src/rdf4cpp/Node.hpp

```diff
@@ -18,6 +18,57 @@ namespace query {
 struct Variable;
 } // namespace rdf4cpp
 
+/**
+ * Options to control the serialization of Nodes/Literals
+ */
+struct NodeSerializationOpts {
+    bool use_short_form; ///< use the short form of the literals, for example: if this option is true "1"^^xsd:integer serializes to 1
+    bool use_prefixes;   ///< use prefixes for known datatypes, for example: if this option is true "1"^^xsd:byte serializes to "1"^^xsd:byte
```

Comment:
state the exact rule here and list the types that are used by default (xsd, rdf).